assignment7.js

```diff
@@ -0,0 +1,9 @@
+function displayCurrentTime() {
+    const d = new Date();
+    const hour = d.getHours();
+    const minute = d.getMinutes();
+    const second = d.getSeconds();
+    const result = `${hour.toString()}:${minute.toString()}:${second.toString()}`;
```

Comment:
why using toString()

Reply:
Fair point. Could have been done without using toString. Just used toString to get the string representation of date.